tests/Makefile

```diff
@@ -1,30 +1,29 @@
 # Make any tests, and possibly run them.
-# A selection of variables are exported from the master Makefile
-# libtap.so and tap.h must be in the local directory or on the system path.
+# A selection of variables are exported from the master Makefile.
 
-# ODIR is realative, so we can use the one from the main Makefile
-
-# As each test will have a main function we need to handle this file by file
-TEST_SOURCES = $(wildcard *_test.cpp)
-TEST_OBJS = $(TEST_SOURCES:%.cpp=$(ODIR)/%.o)
+# As each test will have a main function we need to handle this file by file.
+# We're using the fairly typical convention that any file ending in _test.cpp
+# is a test executable.
+SOURCES = $(wildcard *_test.cpp)
+OBJS = $(TEST_SOURCES:%.cpp=$(ODIR)/%.o)
```

Comment:
It appears `TEST_SOURCES` here and in the line below must be renamed as well. Otherwise, `OBJS` and `TESTS` end up empty.